heidrun/smithsonian.rb

```diff
@@ -378,8 +378,13 @@ def creator?(value)
     # dcterms:language
     #   <language> (not iso-6393 format)
     # TODO: This will need an enrichment to convert to iso-6393
-    language record.field('indexedStructured', 'language').map(&clean_language)
-
+    language :class => DPLA::MAP::Controlled::Language,
+             :each => record.field('indexedStructured', 'language').map(&clean_language)
+             :as => :lang do
+      providedLabel lang
+    #Gretchen added the class mapping on 9/7. Not sure if it is done correctly,
```

Comment:
Do you want to remove those comments at the end? We need to QA the smithsonian mapping anyway since they aren't live in Heidrun yet.